docs/Operators.md

```diff
@@ -5566,46 +5566,164 @@ Other versions of this operator: <a href="Changelog.md#Gemm-1">Gemm-1</a>, <a hr
 #### Examples
 
 <details>
-<summary>notranspose</summary>
+<summary>alpha</summary>
+
+```python
+node = onnx.helper.make_node(
+    'Gemm',
+    inputs=['a', 'b', 'c'],
+    outputs=['y'],
+    alpha=0.5
+)
+a = np.random.ranf([3, 5]).astype(np.float32)
+b = np.random.ranf([5, 4]).astype(np.float32)
+c = np.zeros([1, 4]).astype(np.float32)
+y = 0.5 * np.dot(a, b) + c
```

Comment:
[nit] This is good enough for tests. On the other hand, it'd be nice if you can create a reference implementation of Gemm using numpy and use it in all tests. For example,
```
y = your_gemm_reference_impl(a, b, c, ...)
```

Reply:
I agree, it will be good to have a reference implementation, I'll add one. 👍